lib/checks/base_paths_missing_from_rummager.rb

```diff
@@ -0,0 +1,34 @@
+module Checks
+  class BasePathsMissingFromRummager
+
+    def initialize(checker_db, whitelist)
+      @checker_db = checker_db
+      @whitelist = whitelist
+    end
+
+    # find content_ids present in Publishing API which have a published content item
+    # and for which no base_path is present in Rummager
+
+    # assumption for now: if any content_item was ever published under this content_id,
+    # then we should have it in Rummager; otherwise, we shouldn't care.
+
+    def run_check
+      query = <<-SQL
+      SELECT
+      pac.content_id,
```

Comment:
Indentation in these select statements is inconsistent with other `Check` classes. Let's pick one style and apply throughout. FWIW, I think indenting reads easier.